data/free worlds start.txt

```diff
@@ -1418,6 +1515,68 @@ mission "FW Pirates 3.2B"
 				accept
 
 
+	
+mission "FW Pirates 3.2D"
+	name "Discretion is the Better Part"
+	description "Escape the pirate armada and report to Tomek."
+	priority
+	landing
+	to offer
+        has "FW Pirates 3.2C: active"
+	source Greenrock
+	destination Longjump
+	on fail
+        dialog `You have failed an essential Free Worlds mission. If you want to complete the story line, revert to the autosave or another earlier snapshot of the game.`
+	on offer
+	set "fw pirates: victory"
+	conversation
```

Comment:
Use a `dialog` box here. It's a small amount of text so would look a bit odd in the conversation box and dialog will give a better view of the pirate fleet around the edge.